src/pages/chooseValidity.tsx

```diff
@@ -18,55 +18,89 @@ interface ValidityProps {
     productName: string;
     productPrice: string;
     passengerType: string;
-    daysValid: string;
+    amount: string;
     errors: ErrorInfo[];
     csrfToken: string;
+    timePeriodType: string;
 }
 
 const ChooseValidity = ({
     productName,
     productPrice,
     passengerType,
-    daysValid,
+    amount,
     errors,
+    timePeriodType,
     csrfToken,
 }: ValidityProps): ReactElement => (
     <TwoThirdsLayout title={title} description={description}>
         <CsrfForm action="/api/chooseValidity" method="post" csrfToken={csrfToken}>
             <>
                 <ErrorSummary errors={errors} />
-                <div className={`govuk-form-group ${errors.length ? 'govuk-form-group--error' : ''}`}>
-                    <label htmlFor="validity">
-                        <h1 className="govuk-heading-l" id="choose-validity-page-heading">
-                            How many days is your product valid for?
-                        </h1>
-                    </label>
+                <fieldset className="govuk-fieldset">
```

Comment:
Do we need this fieldset to be linked to the heading via an `aria-describedby` tag?

Reply:
Because it has a legend, it doesn't (advised by Laurence).